### ISSUE TITLE
lambda policy - multiple resource ids could cause issue

### ISSUE THREAD

**Issue by kapilt** [capitalone/cloud-custodian#811]
per traceback on gitter from @jdubs 
```
ot enough arguments for format string: TypeError
Traceback (most recent call last):
File "/var/task/custodian_policy.py", line 4, in run
return handler.dispatch_event(event, context)
File "/var/task/c7n/handler.py", line 77, in dispatch_event
p.push(event, context)
File "/var/task/c7n/policy.py", line 492, in push
return mode.run(event, lambda_ctx)
File "/var/task/c7n/policy.py", line 296, in run
resources = self.resolve_resources(event)
File "/var/task/c7n/policy.py", line 273, in resolve_resources
self.policy.log.info('Found resource ids: %s' % resource_ids)
TypeError: not enough arguments for format string
```

if resource ids is a list then its literal interpolation of multiple elements, we need to wrap resource ids to assert a singleton format string param.

**Comment by scotwk** [comment]
Looks to be fixed in 7196ae0f

**Comment by kapilt** [comment]
yup, thanks
